Source/control.cpp

```diff
@@ -1030,6 +1049,34 @@ void DrawGoldSplit(const Surface &out, int amount)
 	DrawString(out, tempstr, GetPanelPosition(UiPanels::Inventory, { dialogX + 37, 128 }), UiFlags::ColorWhite | UiFlags::PentaCursor);
 }
 
+void DrawGoldWithdraw(const Surface &out, int amount)
+{
+	const int dialogX = 30;
+
+	CelDrawTo(out, GetPanelPosition(UiPanels::Stash, { dialogX, 178 }), *pGBoxBuff, 1);
+
+	constexpr auto BufferSize = sizeof(tempstr) / sizeof(*tempstr);
+
+	CopyUtf8(tempstr, fmt::format(ngettext("How many gold pieces do you want to withdraw? (MAX 5000)","How many gold pieces do you want to withdraw? (MAX 5000)", initialWithdrawGoldValue), initialWithdrawGoldValue), BufferSize);

```

Comment:
Don't format a string with no variables, don't plural translate a string with only one form:
```suggestion
		CopyUtf8(tempstr, _("How many gold pieces do you want to withdraw? (MAX 5000)"), BufferSize);
```